app/stacks/uk-west/back-office/storage-account.tf

```diff
@@ -38,6 +38,15 @@ resource "azurerm_eventgrid_system_topic" "back_office_documents_system_topic" {
   topic_type             = "Microsoft.Storage.StorageAccounts"
 }
 
+# Temporary storage for documents from front office before they're submitted
+resource "azurerm_storage_container" "back_office_submissions_container" {
+  #TODO: Logging
+  #checkov:skip=CKV2_AZURE_21 Logging not implemented yet
+  name                  = "submission-documents"
```

Comment:
Let's call this application-submission-documents because I think we'll create one for appeals eventually too